### PR TITLE
Add version support table to README

### PR DESCRIPTION
**Pull request by luabud**
Document supported versions of Python based in this comment: https://github.com/microsoft/vscode-docs/pull/6979#discussion_r1466647671

### CODE REVIEW

**Comment by luabud** [comment]
@paulacamargo25 any ideas why these tests are failing?

**Comment by paulacamargo25** [comment]
> @paulacamargo25 any ideas why these tests are failing?

yes, i already fix them

File: README.md [hunk 1]
```diff
@@ -33,5 +33,21 @@ If you want to disable the Python Debugger extension, you can [disable this exte
...
+
+Older versions of the Python Debugger extension are available for debugging Python projects that use outdated Python versions like Python 2.7 and Python 3.6. However, it’s important to note that our team is no longer maintaining these extension versions. We strongly advise you to update your project to a supported Python version if possible.
+
+You can reference the table below to find the most recent Python Debugger extension version that offers debugging support for projects using deprecated Python versions.
+
+| Python version | Latest supported Python Debugger extension version |
+| -------------- | -------------------------------------------------- |
+| 2.7            | 2023.1.XXX                                         |
+| 3.6            | 2023.1.XXX                                         |
+| 3.7            | 2023.3.XXX                                         |
```
Comment (paulacamargo25) [inline README.md:46]: Could you also add the version of `debugpy` that each one is using?

Comment (luabud) [reply README.md]: great point!

File: README.md [hunk 2]
```diff
@@ -37,17 +37,16 @@ If you want to disable the Python Debugger extension, you can [disable this exte
 
 Older versions of the Python Debugger extension are available for debugging Python projects that use outdated Python versions like Python 2.7 and Python 3.6. However, it’s important to note that our team is no longer maintaining these extension versions. We strongly advise you to update your project to a supported Python version if possible.
 
-You can reference the table below to find the most recent Python Debugger extension version that offers debugging support for projects using deprecated Python versions.
+You can reference the table below to find the most recent Python Debugger extension version that offers debugging support for projects using deprecated Python versions, as well as the debugpy version that is shipped in each  extension version.
```
Comment (brettcannon) [inline README.md:40]:
```suggestion
You can reference the table below to find the most recent Python Debugger extension version that offers debugging support for projects using deprecated Python versions, as well as the debugpy version that is shipped in each extension version.
```

**Review by paulacamargo25 — APPROVED** [review]

**Pull request merged**